src/lib/data/creatables/slayer.ts

```diff
@@ -31,6 +31,14 @@ export const slayerCreatables: Createable[] = [
 		outputItems: resolveNameBank({ 'Ferocious gloves': 1 }),
 		GPCost: 0
 	},
+	{
+		name: 'Revert hydra leather',
```

Comment:
```suggestion
		name: 'Hydra leather',
```

you arnt reverting the leather you're reverting the gloves.  so just make this `hydra leather`